### PR TITLE
fix short matrix for 2021

### PR DESCRIPTION
**Pull request by kpedro88**
#### PR description:

One-line fix for #27433 (based on comment in https://github.com/cms-sw/cmssw/pull/27644#issuecomment-517689881)

### CODE REVIEW

**Comment by kpedro88** [comment]
please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by zhenhu** [comment]
+1

File: Configuration/PyReleaseValidation/scripts/runTheMatrix.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def runSelected(opt):
                      10024.0, #2017 ttbar
                      10224.0, #2017 ttbar PU
                      10824.0, #2018 ttbar
-                     11624.0, #2021 ttbar
+                     11634.0, #2021 ttbar
```
Comment (slava77) [inline Configuration/PyReleaseValidation/scripts/runTheMatrix.py]: can we make the runner throw if the workflow is not available?

Comment (fabiocos) [reply Configuration/PyReleaseValidation/scripts/runTheMatrix.py]: @slava77 this is a good suggestion, I would manage it independently on this PR

**Comment by fabiocos** [comment]
+1

File: Configuration/PyReleaseValidation/scripts/runTheMatrix.py (hunk 1 above)
Comment (slava77) [reply Configuration/PyReleaseValidation/scripts/runTheMatrix.py]: given some failures in 2023,2024 in some other tests, should we add at least one to the limited set?

**Pull request merged**